lib/influx.rb

```diff
@@ -68,7 +68,7 @@ def find_incidents(start_date = nil, end_date = nil, query_input = nil)
       end_date = Chronic.parse(end_date, :guess => false).last unless end_date.nil?
 
       # If we couldn't parse the given date, use the last 24 hours.
-      end_date = end_date.nil? ? Time.now.to_i : end_date.to_i
+      end_date = (end_date.nil? || end_date.to_i > Time.now.to_i) ? Time.now.to_i : end_date.to_i
```

Comment:
There is a lot happening in that line now. I'd be in favour of wrapping this over 2 lines so the code is more maintain and readable.